### ISSUE TITLE
Report status dialog missing accessible name

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#811]
The report status dialog is contained in a div declared with the HTML:

`<div role="dialog" aria-modal="true" class="modal show" tabindex="-1" aria-labelledby="basic-modal" style="display: block;">`

The id "basic-modal" does not appear to exist. The element that should have an ID that is referenced by aria-labeledby is:

`<h1 class="modal-title css-yefjat" tabindex="-1">`

**Comment by mcking65** [comment]
@ccanash 

The status is still showing as to-do, but there is a PR and the issue appears fixed in the sandbox.

**Comment by ccanash** [comment]
Hi @mcking65 , I  have updated the status and will confirm it was released to production tomorrow.
